README.md

```diff
@@ -424,6 +425,7 @@ The following people have helped making **refined** great:
 * [Vladimir Koshelev](https://github.com/koshelev)
 * [Yuki Ishikawa](https://github.com/rider-yi)
 * [Zainab Ali](https://github.com/zainab-ali)
+* [Michael Thomas](https://github.com/Michaelt293)
```

Comment:
The PR looks good, thanks @Michaelt293! I've only one request: could you please keep this list in alphabetical order? :-)

Reply:
@fthomas Sure thing :)